exporters/common/src/main/java/io/opentelemetry/exporter/internal/marshal/Serializer.java

```diff
@@ -366,6 +395,42 @@ public void serializeRepeatedUInt64(ProtoFieldInfo field, DynamicPrimitiveLongLi
     writeEndRepeatedVarint();
   }
 
+  /** Serializes a {@code repeated int64} field. */
+  public void serializeRepeatedInt64(ProtoFieldInfo field, long[] values) throws IOException {
```

Comment:
This appears unused.

Reply:
Oops. Vestage of an experiment in using arrays rather than lists to avoid instantiating small objects for each element I think. Removed.